src/overlays/actors/ovl_En_Fw/z_en_fw.h

```diff
@@ -3,12 +3,47 @@
 
 #include <ultra64.h>
 #include <global.h>
+#include "../ovl_En_Fd/z_en_fd.h"
 
 struct EnFw;
 
+typedef void (*EnFwActionFunc)(struct EnFw* this, GlobalContext* globalCtx);
+
+typedef struct {
```

Comment:
This is the same structure as EnFdPart. Since you're including its header already, it should be fine to just reuse its particle struct.

Reply:
They are the same in terms of their members, but the represent different things.

Reply:
Yes, I can see now how that wouldn't work. In that case, my suggestion is to switch from `used` to `type`, to be consistent both with the other struct and with other actors that use the same particle code templates.

Reply:
agree